apps/server/src/modules/user/legacy/controller/dto/class.response.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { ClassResponse } from './class.response';
```

Comment:
These tests for the response classes are not necessary. You can delete the files.

Reply:
I created them to achieve 100% coverage

Reply:
You should achieve that through the api-tests. You just need to create a case where all response classes are instantiated.